specs/dwritecore/DWriteCore.md

```diff
@@ -78,33 +78,275 @@ with interoperability issues.
+`DWRITE_FACTORY_TYPE_ISOLATED`, but is more locked down in two ways. First, it only caches data in-process, and
 will not attempt to either read from or write to a cross-process font cache or persisted cache file. Second,
 a restricted factory only enumerates well-known fonts in the system font collection. The well-known-fonts
 restriction enables DWriteCore to efficiently construct the system font collection without depending on a
 cross-process or persistent cache. An example use case for the new factory type would be a web browser
 rendering process with very limited permissions.
 
-A _bitmap render target_ (IDWriteBitmapRenderTarget) is an API object that encapsulates a system memory bitmap
+# Text Rendering
+
+A _bitmap render target_ (`IDWriteBitmapRenderTarget`) is an API object that encapsulates a system memory bitmap
 and enables rendering glyphs to the bitmap. On Windows, the object encapsulates a GDI memory DC with a GDI
 device-dependent bitmap (DIB) selected into it. However, on non-Windows platforms, there is no GDI DC,
 and the bitmap is just a system memory array. DWriteCore introduces a new version of the bitmap render target
 interface that an application can use to access the bitmap data without going through GDI.
 
+# Font Selection
+
+The new `IDWriteFontSet4` interface exposes new methods for selecting fonts from a font set. The new methods
+make it possible to transition to the _typographic font family model_ while maintaining compatibility with
+existing applications, documents, and fonts.
+
+Font selection (sometimes called font matching or font mapping) is the process of selecting the available
+fonts that best match input parameters passed in by an application. The input parameters are sometimes
+referred to collectively as a _logical font_. A font selection algorithm matches the logical font (the
+"font you want") to an available _physical font_ (a "font you have").
+
+A _font family_ is a named group of fonts that share a common design but may differ in attributes such as
+weight. A _font family model_ defines what attributes may be used to differentiate fonts within a family.
+The new _typographic font family model_ has many advantages over the two previous font family models used
+on Windows. Unfortunately, changing font family models creates opportunities for confusion and compatibility
+issues. The methods exposed by the `IDWriteFontSet4` interface implement a hybrid approach that offers the
+advantages of the typographic font family model while mitigating compatibility issues.
+
+The following sections compare the older font family models with the typographic font family model,
+explain compatibility challenges posed by changing font family models, and finally explains how those
+challenges can be overcome by using the `IDWriteFontSet4` methods.
+
+## RBIZ Font Family Model
+
+The font family model used by GDI is sometimes called the "four-font model" or "RBIZ" model. Each font
```

Comment:
To be accurate, this wasn't GDI's family model. For example, GDI can support a family with multiple weights (not just RB) provided name IDs 1 and 2 are set accordingly in the fonts. RBIZ was a model assumed early on by apps (Word) and it was compatibility with apps that led font developers to assume name IDs 1 and 2, and GDI, *require* the RBIZ model.

I'd word this, "... used by GDI apps..."

Reply:
Yes, I understand RBIZ is not actually GDI's font family model, but it's become the de facto model in the GD application ecosystem. I'm rewording this section to hopefully make that more clear.